server.go

```diff
@@ -174,6 +195,10 @@ var startupHooks []func()
 // This can be useful when you need to establish connections to databases or third-party services,
 // setup app components, compile assets, or any thing you need to do between starting Revel and accepting connections.
 //
-func OnAppStart(f func()) {
-	startupHooks = append(startupHooks, f)
+func OnAppStart(f func(), order ...int) {
+	o := 1
```

Comment:
I think the default value should be 5 and we have some sort of guideline as to the range of values to be used.
ie  Default / normal is 5, high / first is 1, low / last is 10. 
So people know that revel builtin startuphooks will run at 5, if they want something before or after it should be higher or lower.